app/services/hyrax/workflow/notification_service.rb

```diff
@@ -51,10 +51,10 @@ def recipients(notification)
         case entity.workflow_state.name
         when 'pending_review'
           # notify the managers to review the deposit (CC depositors)
-          {
-            to: entity.workflow.permission_template.agent_ids_for(access: 'manage',  agent_type: 'user'),
-            cc: entity.workflow.permission_template.agent_ids_for(access: 'deposit', agent_type: 'user')
-          }
+          permissions = entity.workflow.permission_template
+          managers = Array.wrap(permissions.agent_ids_for(access: 'manage', agent_type: 'user'))
```

Comment:
I'm not sure we need the `Array.wrap` calls here; `agent_ids_for` should always return an array.